.travis.yml

```diff
@@ -33,6 +34,24 @@ jobs:
       after_script:
         - ./cc-test-reporter after-build --exit-code $TRAVIS_TEST_RESULT
   
+    - name: "regression test using python-behave thread 1"
+      before_install:
+        - /bin/bash $TRAVIS_BUILD_DIR/test/docker_scripts.sh before_install
```

Comment:
I guess that you have added this `thread` thing to run the tests in parallel but it doesn't look like really scalable...